src/classes/Auth/Auth.ts

```diff
@@ -44,12 +44,17 @@ export class Auth extends PassageBase {
             }
 
             const {
-                payload: { sub: userId },
+                payload: { sub: userId, aud },
             } = await jwtVerify(jwt, this.jwks);
 
             if (!userId) {
                 throw new PassageError('Could not validate auth token.');
             }
+            if (Array.isArray(aud)) {
```

Comment:
Just confirming that the array check is used in lieu of the jwtVerify [audience option](https://github.com/panva/jose/blob/main/docs/jwt/verify/interfaces/JWTVerifyOptions.md#audience) for that backwards compatibility of only verifying new tokens, otherwise it would fail when comparing embedded tokens to the app id

Reply:
Correct, I want this package to still be able to validate old embedded tokens without failing.

Reply:
My other concern with that built-in audience check is the aud array isn't only the app id, it also includes the auth origin which isn't data available at the time of jwt validation (unless we hit the app info endpoint):
<img width="576" alt="image" src="https://github.com/user-attachments/assets/66fae7e5-881e-4a3d-aa1a-55085e53a6e4">